x-pack/plugins/observability_solution/dataset_quality/common/api_types.ts

```diff
@@ -56,8 +56,13 @@ export const degradedDocsRt = rt.type({
 export type DegradedDocs = rt.TypeOf<typeof degradedDocsRt>;
 
 export const dataStreamDetailsRt = rt.partial({
-  createdOn: rt.number,
+  createdOn: rt.union([rt.null, rt.number]), // rt.null is needed because `createdOn` is not available on Serverless
```

Comment:
Do we hide this row, in the dataset details section, on serverless?

Reply:
Yes, we show `-` for empty value.